--- origsrc/mate-panel-1.20.1/libmate-panel-applet/mate-panel-applet.c	2018-03-22 06:49:30.000000000 -0500
+++ src/mate-panel-1.20.1/libmate-panel-applet/mate-panel-applet.c	2018-04-23 10:23:25.251987400 -0500
@@ -2362,7 +2362,7 @@ mate_panel_applet_get_object_path (MateP
 	return applet->priv->object_path;
 }
 
-G_MODULE_EXPORT GtkWidget *
+GtkWidget *
 mate_panel_applet_get_applet_widget (const gchar *factory_id,
                                 guint        uid)
 {
