--- origsrc/mate-panel-1.6.0/libmate-panel-applet/Makefile.am	2013-03-31 16:36:43.000000000 -0500
+++ src/mate-panel-1.6.0/libmate-panel-applet/Makefile.am	2013-04-04 02:47:06.184232900 -0500
@@ -118,7 +118,7 @@ built_introspection_sources =	\
 
 MatePanelApplet-4.0.gir: libmate-panel-applet-4.la
 MatePanelApplet_4_0_gir_INCLUDES = GObject-2.0 Gtk-2.0 Gio-2.0
-MatePanelApplet_4_0_gir_CFLAGS = $(AM_CPPFLAGS)
+MatePanelApplet_4_0_gir_CFLAGS = $(AM_CPPFLAGS) -I$(srcdir)
 MatePanelApplet_4_0_gir_LIBS = libmate-panel-applet-4.la
 MatePanelApplet_4_0_gir_FILES = $(addprefix $(srcdir)/,$(introspection_sources)) $(addprefix $(builddir)/,$(built_introspection_sources))
 # When requiring gobject-introspection 0.9.5, use this:
